vumi/transports/vumi_bridge/vumi_bridge.py

```diff
@@ -323,10 +231,21 @@ def handle_raw_inbound_event(self, request):
             msg = TransportEvent(_process_fields=True, **to_kwargs(data))
             yield self.handle_inbound_event(msg)
             request.finish()
+            if msg.payload["event_type"] == "ack":
+                self.update_status(
+                    status='ok', component='sent-by-vumi-go',
+                    type='vumi_go_sent', message='Sent by Vumi Go')
+            else:
```

Comment:
Events could also be delivery reports, so we should do something like `elif msg.payload["event_type"] == "nack"` here.